core/templates/domain/exploration/InteractionObjectFactorySpec.ts

```diff
@@ -775,14 +780,17 @@ describe('Interaction object factory', () => {
 
   it('should fully cover constructing customization arguments for all ' +
      'interactions', () => {
-    Object.keys(INTERACTION_SPECS).forEach(interactionId => {
+    type Keys = keyof typeof INTERACTION_SPECS;
```

Comment:
Can you please look into the codebase if there is already keys type of `INTERACTION_SPECS`? Or if not please use better name for this type.

Reply:
done, could not find a type for keys of INTERACTION_SPECS, so made a new type and added it to the top, so it can be used elsewhere